homework/Repository.js

```diff
@@ -12,9 +12,29 @@ class Repository {
    * Render the repository info to the DOM.
    * @param {HTMLElement} container The container element in which to render the repository.
    */
+  // container = ul
   render(container) {
```

Comment:
A two-column HTML table instead of an unordered list would have produced a bit sleeker UI, but okay, this works too.

Reply:
Thanks, U are right! I already used the table when I was in class 20, so I thought it could be better if I try it in another way which is an unordered list :)